scripts/cphd/copyright_update.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+#
+# =========================================================================
+# This file is part of cphd-python 
+# =========================================================================
+# 
+# (C) Copyright 2004 - 2019, MDA Information Systems LLC
+#
+# cphd-python is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public 
+# License along with this program; If not, 
+# see <http://www.gnu.org/licenses/>.
+#
+
+#
+# Script to edit copyright dates and company of files
+#
+
+import sys
+import os
```

Comment:
How much work would it be to generalize this to any module, not just cphd?

Reply:
If I remember correctly, I tried to make it generalizable. I put in a bit of additional error checking specific to the format of our files, but I can get rid of that and explicitly state the assumptions the code makes